.github/actions/validate-docs-links/src/checker.ts

```diff
@@ -168,7 +170,7 @@ function validateInternalLink(errors: Errors, href: string): void {
 
   // check if doc page exists
   const foundPage = documentMap.get(link.replace(/^\/+/, ''))
-  
+
```

Comment:
Not super important, but I guess it's best to avoid unnecessary blank lines (unless this is bringing it up to date with the formatting we use in the main site repo).

Reply:
It's actually [common practice](https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file) to add a blank line at the end of source files. Different linters and syntax guides recommend this.

It's so common that my IDE automatically inserts blank lines at the end of files, hence the addition. Sorry 😅